spec/models/rating_spec.rb

```diff
@@ -19,9 +18,18 @@
   end
 
   it "is invalid if score is not between 0 - 4" do
-    rating = build(:rating, score: 5)
+    rating = build(:rating, score: 5, user: create(:user))
```

Comment:
It almost definitely isn't necessary and probably a result of when I was trying to figure out that duplicate creation issue.